tests/unit/Functions/Validator/HeadersTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Unit\Functions\Validator;
+
+use Appwrite\Functions\Validator\Headers;
+use PHPUnit\Framework\TestCase;
+
+class HeadersTest extends TestCase
+{
+    protected ?Headers $object = null;
+
+    public function setUp(): void
+    {
+        $this->object = new Headers();
+    }
+
+    public function testValues(): void
```

Comment:
Validator accepts `param mixed $value`. Lets try to pass some invaid value like string, number, boolean or null. Ensure they return false too, as they are not valid

Reply:
Is `1234 => 'value',` a valid header?

Reply:
It is not, we can have assertion for that. But I meant `$headres = 123` or `$headers = false`. When header isn't array.